packages/client/hmi-client/src/components/workflow/ops/simulate-ciemss/tera-simulate-ciemss-drilldown.vue

```diff
@@ -126,30 +126,6 @@
 				</template>
 			</tera-slider-panel>
 		</tera-drilldown-section>
-
-		<!-- Notebook -->
-		<tera-drilldown-section :tabName="DrilldownTabs.Notebook" class="notebook-section">
-			<tera-notebook-jupyter-input
-				:kernel-manager="kernelManager"
-				:context-language="'python3'"
-				@llm-output="(data: any) => processLLMOutput(data)"
-				@llm-thought-output="(data: any) => llmThoughts.push(data)"
-				@question-asked="updateLlmQuery"
-			>
-				<template #toolbar-right-side>
-					<Button label="Run" size="small" icon="pi pi-play" @click="runCode" :disabled="isEmpty(codeText)" />
-				</template>
-			</tera-notebook-jupyter-input>
-			<v-ace-editor
-				v-model:value="codeText"
-				@init="initializeAceEditor"
-				lang="python"
-				theme="chrome"
-				style="flex-grow: 1; width: 100%"
-				class="ace-editor"
-			/>
```

Comment:
Double check with pascale on this. We can

1. Remove the notebook/jupyter altogether, or
2. Still have the notebook, but have it showing the request payload that we send to pyciemss-service instead.

For Funman we took the 2nd approach, so pascale may want to keep things consistent.

Reply:
After quick chat in design channel I believe Pascale is fine with just removing this entirely.

Main reason:
`
As they go through pyciemss-service the payloads are not directly 1:1 with pyciemss calls. The user would also need to be familiar with pyciemss to gain any benefit even if they did match 1:1
`